tree/ntuple/src/RFieldMeta.cxx

```diff
@@ -71,7 +71,25 @@ TEnum *EnsureValidEnum(std::string_view enumName)
    return e;
 }
 
-std::string BuildSetTypeName(ROOT::RSetField::ESetType setType, const ROOT::RFieldBase &innerField)
+std::string GetTypeList(std::span<std::unique_ptr<ROOT::RFieldBase>> itemFields, bool useTypeAliases)
```

Comment:
Consider adding a short summary of the behavior (Intend of the use of the result, reach of the list).